src/app/containers/AssignmentEditModal.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import {
+    Button,
+    Glyphicon
+} from 'react-bootstrap';
+import AssignmentTemplateEditForm from './AssignmentTemplateEditForm';
+import ModalWrapper from './ModalWrapper';
+
+export interface AssignmentEditModalProps {
+    templateId: number;
+}
+
+export default class AssignmentEditModal extends React.Component<AssignmentEditModalProps>{
+    render() {
+        const { templateId } = this.props;
+        
+        return (
+            <div>                
+                <ModalWrapper
```

Comment:
Yea, really liking this pattern.  Looks good.  since you only have the ModalWrapper, you can get rid of the outer `<div>`